index.js

```diff
@@ -1,10 +1,33 @@
 module.exports = (robot) => {
-  // Your code here
-  console.log('Yay, the app was loaded!')
+  robot.on(['pull_request.opened', 'issues.opened'], async context => {
+    const params = newActionParams(context)
+    const result = await context.github.issues.addLabels(params)
+    robot.log(result)
+    return result
+  })
 
-  // For more information on building apps:
-  // https://probot.github.io/docs/
+  function newActionParams(context) {
 
-  // To get your app running against GitHub, see:
-  // https://probot.github.io/docs/development/
+    params = {
+      owner: context["payload"]["repository"]["owner"]["login"],
+      repo: context["payload"]["repository"]["name"],
+      number: issue_or_pr_number(context),

```

Comment:
I think javascript uses `mixedCase` rather than `snake_case`.

Reply:
✨ updated in b654dd6